src/videojs.ads.js

```diff
@@ -113,12 +114,14 @@ var
           updateEnded = function() {
             ended = true;
           };
-        player.currentTime(snapshot.currentTime);
```

Comment:
If this is a fact of life with MSE, I wonder if there ought to be some detection of the type of source in the player or an explicit option that flags the player as using a segmented format.